.gitignore

```
@@ -30,6 +31,9 @@ spec/decidim_dummy_app/
 /public/packs/
 /public/packs-test/
 /public/decidim-packs/
+/public/sw.js
+/public/sw.js.map
+
```

must add the other variants?
```suggestion
/public/sw.js
/public/sw.js.br
/public/sw.js.gz
/public/sw.js.map
/public/sw.js.map.br
/public/sw.js.map.gz
```